wcc-contentful/lib/wcc/contentful/simple_client/cdn.rb

```diff
@@ -81,13 +81,16 @@ def content_types(query = {})
   def sync(sync_token: nil, **query, &block)
     return sync_old(sync_token: sync_token, **query) unless block_given?
 
-    sync_token =
+    query = {
+      # override default locale for sync queries
+      locale: nil
```

Comment:
Here's one case where our VCRs will change.  The URL will go from `/sync?locale=*` to just `/sync`.